### PR TITLE
feat(Task-14): As a User, I want to get a suggestion list of cities when I write on a search input.

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

File: src/app/shared/components/displayable-list/displayableList.controller.js [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
+
 export default class DisplayableListController {
-    constructor(routingFunctionsService) {
+    constructor(routingFunctionsService, $ngRedux) {
         this.routingFunctionsService = routingFunctionsService;
+        this.$ngRedux = $ngRedux;
     }
 
     search(city) {
+        console.log(city)
```
Comment (gbtec-ext-jorgevillar) [inline src/app/shared/components/displayable-list/displayableList.controller.js]: Remove log

File: src/app/shared/services/geoNames.js [hunk 2]
```diff
@@ -5,28 +5,45 @@ export default class GeoNames {
...
+
         const nearbyParams = {
             params: {
-                lat: 0,
-                lng: 0,
+                lat: city.lat,
+                lng: city.lng,
                 cities: cities,
                 radius: radius,
                 maxRows: maxRows,
```
Comment (gbtec-ext-jorgevillar) [inline src/app/shared/services/geoNames.js:49]: You can write these three in the ES6

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]
Fix search button

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Review by eloydotco — APPROVED** [review]

**Comment by jLagosCarrera** [comment]
#16 solved here

**Comment by jLagosCarrera** [comment]
#15 solved here aswell

**Pull request merged**